packages/graphics/glmark2/package.mk

```diff
@@ -2,31 +2,19 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="glmark2"
-PKG_VERSION="7632c2e2677ef01e44af758823c2fffcced22524"
-PKG_SHA256="c0fb0fb2f62d05f229466a4c1cbb902816595b0a5ed860de46f763533c515e6c"
+PKG_VERSION="784aca755a469b144acf3cae180b6e613b7b057a"
+PKG_SHA256="787dd7de05c795c7c42694e648b5c6418e4b395fc460ea349d1d35d493469356"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/glmark2/glmark2"
 PKG_URL="https://github.com/glmark2/glmark2/archive/$PKG_VERSION.tar.gz"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="glmark2 is an OpenGL 2.0 and ES 2.0 benchmark"
-PKG_TOOLCHAIN="manual"
+PKG_TOOLCHAIN="meson"
```

Comment:
should be able to drop this line completely

Reply:
👍
```
$ PROJECT=Generic ARCH=x86_64 IMAGE_SUFFIX=rudi s/build glmark2
UNPACK      glmark2
BUILD      glmark2 (target)
    TOOLCHAIN      meson (auto-detect)
Executing (target): meson --prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --libdir=/usr/lib --libexecdir=/usr/lib --localstatedir=/var --buildtype=plain -Dstrip=true --cross-file=/home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/glmark2-784aca755a469b144acf3cae180b6e613b7b057a/.x86_64-libreelec-linux-gnu/meson.conf -Dflavors=drm-gl /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/glmark2-784aca755a469b144acf3cae180b6e613b7b057a
```